project-p3-forms-master/code/index.html

```diff
@@ -0,0 +1,124 @@
+            <option value="Gesundheit">Gesundheit</option>
+            <option value="Bildung">Bildung</option>
+            <option value="Entwicklungszusammenarbeit">Entwicklungszusammenarbeit</option>
+            <option value="Marketing">Marketing</option>
+            <option value="Tourismus">Tourismus</option>
+            <option value="IT/Coding">IT/Coding</option>
+            <option value="Handwerk">Handwerk</option>
+            <option value="Gastronomie">Gastronomie</option>
+            <option value="Detailhandel">Detailhandel</option>
+            <option value="Banken">Banken</option>
+            <option value="Versicherung">Versicherung</option>
+            <option value="Journalismus">Journalismus</option>
+            <option value="Asyl/Migration">Asyl/Migration</option>
+            <option value="Andere">Andere</option>
+          </select>
+          <select name="branche">
+            <option value="Gesundheit">Gesundheit</option>
+            <option value="Bildung">Bildung</option>
+            <option value="Entwicklungszusammenarbeit">Entwicklungszusammenarbeit</option>
+            <option value="Marketing">Marketing</option>
+            <option value="Tourismus" selected>Tourismus</option>
+            <option value="IT/Coding">IT/Coding</option>
+            <option value="Handwerk">Handwerk</option>
+            <option value="Gastronomie">Gastronomie</option>
+            <option value="Detailhandel">Detailhandel</option>
+            <option value="Banken">Banken</option>
+            <option value="Versicherung">Versicherung</option>
+            <option value="Journalismus">Journalismus</option>
+            <option value="Asyl/Migration">Asyl/Migration</option>
+            <option value="Andere">Andere</option>
+          </select>
+          <select name="branche">
+            <option value="Gesundheit">Gesundheit</option>
+            <option value="Bildung">Bildung</option>
+            <option value="Entwicklungszusammenarbeit">Entwicklungszusammenarbeit</option>
+            <option value="Marketing">Marketing</option>
+            <option value="Tourismus">Tourismus</option>
+            <option value="IT/Coding">IT/Coding</option>
+            <option value="Handwerk" selected>Handwerk</option>
+            <option value="Gastronomie">Gastronomie</option>
+            <option value="Detailhandel">Detailhandel</option>
+            <option value="Banken">Banken</option>
+            <option value="Versicherung">Versicherung</option>
+            <option value="Journalismus">Journalismus</option>
+            <option value="Asyl/Migration">Asyl/Migration</option>
+            <option value="Andere">Andere</option>
+          </select>
+        </label>
+        <label>
+          Welche? 
+          <input type="text" placeholder="Nur ausfüllen, wenn du bei Branche Andere gewählt hast"> <!-- nur required wenn das Feld "Andere" angeklickt, Validation JavaScript-->
+        </label> 
```

Comment:
For this to only show in some cases, you need some JavaScript to render this input-field conditionally. Hopefully, you can get back to this project after the first JavaScript project when you've learned how to manipulate the DOM 🤩

Reply:
Still trying to get the hang of it. It will take some more time to figure it all out ;)